build.gradle.kts

```diff
@@ -120,6 +121,15 @@ subprojects {
 
 allprojects {
   plugins.withId("com.vanniktech.maven.publish.base") {
+    configure<PublishingExtension> {
+      // For the Gradle plugin's tests.
```

Comment:
If it's only for tests, should this be only applied during tests then to prevent accidental test publishing in CI?